### ISSUE TITLE
Helm upgrade error: An error occurred: t is null.

### ISSUE THREAD

**Issue by dud225** [kubeapps/kubeapps#4394]
### Description:

I've installed [rbac-manager](https://rbac-manager.docs.fairwinds.com) out of Kubeapps and I'm unable to upgrade it from Kubeapps.

### Steps to reproduce the issue:

1. Install rbac-manager manually:
```
helm repo add fairwinds-stable https://charts.fairwinds.com/stable
helm install fairwinds-stable/rbac-manager --name rbac-manager --namespace rbac-manager
```
2. Create a AppRepository resource pointing to the Fairwinds repository:
```
apiVersion: kubeapps.com/v1alpha1
kind: AppRepository
metadata:
  name: fairwinds
  namespace: kubeapps
spec:
  filterRule:
    jq: .name == $var0
    variables:
      $var0: rbac-manager
  syncJobPodTemplate:
    spec:
      securityContext:
        runAsUser: 1001
  type: helm
  url: https://charts.fairwinds.com/stable
```
3. Upgrade rbac-manager from Kubeapps

### Describe the results you received:

![Capture d’écran de 2022-03-08 09-39-19](https://user-images.githubusercontent.com/17550475/157206674-47e6aa77-a338-4e93-adcc-f181152391ef.png)
![Capture d’écran de 2022-03-08 09-39-31](https://user-images.githubusercontent.com/17550475/157206666-35d43d60-de63-4898-8d82-2c0318c4a6ca.png)

### Version of Helm, Kubeapps and Kubernetes:

- Output of `helm version`:

```
$ microk8s.helm3 version
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/2948/credentials/client.config
version.BuildInfo{Version:"v3.5.0", GitCommit:"32c22239423b3b4ba6706d450bd044baffdcf9e6", GitTreeState:"clean", GoVersion:"go1.15.6"}
```

- Output of `helm list -n <kubeapps-namespace> <kubeapps-release-name>`:

```
$ microk8s.helm3 list -n kubeapps 
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /var/snap/microk8s/2948/credentials/client.config
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
kubeapps        kubeapps        3               2022-03-07 10:07:07.431744879 +0000 UTC deployed        kubeapps-7.8.7  2.4.3
```

- Output of `kubectl version`:

```
$ microk8s.kubectl version
Client Version: version.Info{Major:"1", Minor:"23+", GitVersion:"v1.23.3-2+d441060727c463", GitCommit:"d441060727c4632b67d09c9118a36a8590308676", GitTreeState:"clean", BuildDate:"2022-01-26T22:01:35Z", GoVersion:"go1.17.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"23+", GitVersion:"v1.23.3-2+d441060727c463", GitCommit:"d441060727c4632b67d09c9118a36a8590308676", GitTreeState:"clean", BuildDate:"2022-01-26T21:57:05Z", GoVersion:"go1.17.6", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by absoludity** [comment]
May be related to #4352

**Comment by castelblanque** [comment]
> May be related to https://github.com/kubeapps/kubeapps/issues/4352

Just tested it with both Kubeapps v2.4.3 and current development.
Confirmed that it is a duplicate of #4352 and it will be solved with #4355.
Reason behind is that there are no values provided to the installation.
